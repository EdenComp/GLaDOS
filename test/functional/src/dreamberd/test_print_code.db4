print("fn test(){return \"hello\";}print(test());");