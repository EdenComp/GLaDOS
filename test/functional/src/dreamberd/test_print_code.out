fn test(){return "hello";}print(test());